rest-service/manager_rest/rest/rest_utils.py

```diff
@@ -339,6 +339,8 @@ def update_deployment_dependencies_from_plan(deployment_id,
         for creator, target in new_dependencies.items()
         if dep_plan_filter_func(creator)
     }
+    g = get_recursive_dependencies(storage_manager)
```

Comment:
Let's change it to a more informative name, e.g. `recursive_dependencies` or anything else that you think fits here.